docs/v1/P4Runtime-Spec.mdk

```diff
@@ -1595,6 +1595,12 @@ The `ActionProfile` message includes the following fields:
   Action Profile can hold, or, in the case of an Action Selector, the maximum
   sum of all member weights across all selector groups.
 
+* `max_num_members`, an `int64` representing the maximum number of member
+  entries that an Action Profile or Action Selector can hold. For Action
+  Profiles, this should always be equal to the size. PSA programs can use the
```

Comment:
Nit: maybe swap these two sentence? ("For Action Profiles ..." & "PSA programs")

Reply:
Is `@max_num_members` specific to PSA programs? We're using v1model and are planning to use this, no?

Reply:
Done on the nit. For PSA programs, I just used the same wording as in `max_group_size`. I'm honestly not sure why we talk about how to specify it in PSA programs in this part of the spec at all, though a significant portion of the section talks about PSA in various ways. 

WDYT?

Reply:
Done in other PR #386, to be clear.